canonical-web-and-design/manager.assets.ubuntu.com
# Migrate project to Flask

Migrate from django to flask.

# QA
- Set a `.env.local` as described in the README
- `./run`
- Make sure everything still works

Fixes https://github.com/canonical-web-and-design/base-squad/issues/758
https://github.com/canonical-web-and-design/base-squad/issues/759
